qiskit_machine_learning/runtime/torch_runtime_client.py

```diff
@@ -0,0 +1,495 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The Qiskit Machine Learning Torch Runtime Client"""
+
+import base64
+from typing import Any, Callable, Dict, Optional, Union
+
+import dill
+
+from qiskit.exceptions import MissingOptionalLibraryError
+from qiskit.exceptions import QiskitError
+from qiskit.providers import Provider
+from qiskit.providers.backend import Backend
+from qiskit_machine_learning.runtime.hookbase import HookBase
+
+try:
+    from torch import Tensor
+    from torch.nn import Module, MSELoss
+    from torch.nn.modules.loss import _Loss
+    from torch.optim import Optimizer
+    from torch.utils.data import DataLoader
+except ImportError:
+
+    class DataLoader:  # type: ignore
```

Comment:
Yes, I get a mypy error like this `Name "DataLoader" already defined (possibly by an import)`.
To avoid it, I copied the following existing code. 
https://github.com/Qiskit/qiskit-machine-learning/blob/1baf45cde9e7613c25ad6df831ef5795e479183d/qiskit_machine_learning/algorithms/objective_functions.py#L24
It seems currently we need to add `#type: ignore` to avoid  this error.
https://github.com/python/mypy/issues/1153